src/ucp/rndv/rndv.c

```diff
@@ -266,8 +269,6 @@ ucp_rndv_reg_send_buffer(ucp_request_t *sreq, const ucp_request_param_t *param)
         if (status != UCS_OK) {
             return status;
         }
-
-        sreq->flags |= UCP_REQUEST_FLAG_RKEY_INUSE;
```

Comment:
why removed?

Reply:
it's moved to the place where `rkey` is actually packed

Reply:
yep, this is indeed better.